README.md

```diff
@@ -27,6 +27,29 @@ $ pipenv install
     * `$ pipenv run python blockchain.py` 
     * `$ pipenv run python blockchain.py -p 5001`
     * `$ pipenv run python blockchain.py --port 5002`
+    
+## Docker
+
```

Comment:
Let's add the following to show that running it in Docker is entirely optional:

```
If you'd like to use Docker, you can follow these instructions... etc.
```

Reply:
Good point - I have updated the readme to reflect this.